docs/distro-compat.md

```diff
@@ -33,9 +33,20 @@ methods supported, and any other requirements:
 
 -   "Kernel upgrade" means a kernel upgrade may be required (Sysbox requires a fairly new kernel). See [below](#kernel-upgrade-procedures) for more.
 
-- "EOL" releases refer to those that are being deprecated by its distro vendor as
+-   "EOL" releases refer to those that are being deprecated by its distro vendor as
 part of their release scheduling process.
 
+
+***
+#### ** --- Sysbox-EE Feature Highlight --- **
```

Comment:
Rather than add this paragraph here, I would remove it and instead add the following for the "Other" column in the distro table above:

"Sysbox-EE only; see [here](user-guide/install-flatcar.md) for details."

Reply:
makes sense